lib/fizzy/cxx23/utility.hpp

```diff
@@ -0,0 +1,18 @@
+// Fizzy: A fast WebAssembly interpreter
+// Copyright 2022 The Fizzy Authors.
+// SPDX-License-Identifier: Apache-2.0
+
+#pragma once
+
+#include <type_traits>
+
+namespace fizzy
+{
+/// C++14 version of the proposed C++23 feature
+/// https://www.open-std.org/jtc1/sc22/wg21/docs/papers/2021/p1682r3.html
```

Comment:
I'd prefer a link to https://en.cppreference.com/w/cpp/utility/to_underlying, it's more readable